### PR TITLE
#[#161711202] Put email and Password in environmental variables

### PR DESCRIPTION
**Pull request by johnkegz**
#### What does this PR do?
Comparing ch-put_keys with develop
#### Description of Task to be completed?
putting environmental variables
#### How should this be manually tested?
create a .env file in the root of your project
put the variables as shared
source the .env file in the terminal
and then you are set.
#### Any background context you want to provide?
Security of the API where passwords are not known
#### What are the relevant pivotal tracker stories?
#161711202

### CODE REVIEW

**Comment by bozicschucky** [comment]
This PR looks good to me.
It should be merged so that I can add my fix asap.

File: authors/urls.py [hunk 1]
```diff
@@ -42,13 +38,13 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
```
Comment (joshNic) [inline authors/urls.py:40]: looks good although there is unnecessary space (pep8)

File: authors/urls.py [hunk 2]
```diff
@@ -42,13 +38,13 @@
...
 
-    path('swagger(P<format>.json|.yaml)',
-         schema_view.without_ui(cache_timeout=0), name='schema-json'),
     path('documentation/', schema_view.with_ui('swagger',
                                          cache_timeout=0), name='schema-swagger-ui'),
+
     path('redoc/', schema_view.with_ui('redoc',
                                        cache_timeout=0), name='schema-redoc'),
```
Comment (joshNic) [inline authors/urls.py:46]: looks good although there is unnecessary space (pep8)

File: authors/urls.py [hunk 3]
```diff
@@ -15,15 +15,11 @@
...
-from rest_framework_swagger.views import get_swagger_view
-schema_view = get_swagger_view(title="Author's Haven REST API")
+
 
 from rest_framework.permissions import AllowAny
 from drf_yasg.views import get_schema_view
 from drf_yasg import openapi
-from rest_framework_swagger.views import get_swagger_view
-schema_view = get_swagger_view(title="Author's Haven REST API")
-
```
Comment (joshNic) [inline authors/urls.py:23]: looks good although there is too much space (pep8)

File: authors/urls.py [hunk 4]
```diff
@@ -15,15 +15,11 @@
 """
 from django.urls import include, path
 from django.contrib import admin
-from rest_framework_swagger.views import get_swagger_view
-schema_view = get_swagger_view(title="Author's Haven REST API")
+
```
Comment (joshNic) [inline authors/urls.py:19]: looks good although there is unnecessary space (pep8)

**Comment by emukungu** [comment]
Could you add the secret key to the os as well? Seems you didn't add it in your code

File: authors/urls.py (hunk 2 above)
Comment (johnkegz) [reply authors/urls.py:46]: ok let me format it

**Comment by johnkegz** [comment]
@emukungu let put that as well

**Comment by johnkegz** [comment]
@emukungu @joshNic  @bozicschucky  @cdvx, please review

**Comment by bozicschucky** [comment]
Looks good.

**Review by cdvx — COMMENTED** [review]
Looks good, though there should be a new line the end of each file @johnkegz 

try creating a file `.editorconfig` and add everything in the following text file.
Also add it to the `.gitignore` 
It will help automate the process, in the subsequent changes maybe

**Pull request merged**